tests/unit/test_db.py

```diff
@@ -42,7 +42,6 @@ def setUp(self):
         # Define some relations.
         self.rel_id = self.harness.add_relation(RELATION_NAME, "application")
         self.harness.add_relation_unit(self.rel_id, "application/0")
-        self.harness.add_relation_unit(self.rel_id, self.unit)
```

Comment:
Causes warnings when executing the unit tests.

Reply:
Just curios: is it a relation to itself? o_0

Reply:
I guess it was added to have both a local and remote units in the relation, but the harness method seems to be for simulating a remote unit only.